examples/eventgrid/index.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
```

Comment:
i just realized, but i don't think i've been seeing in your PRs the updates to our testing code to try to actually ensure these build and deploy.  can you update examples_tests.go to do that?  Thanks!

Reply:
Great point! Will make sure they are all there.